static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -36,6 +36,6 @@ repo_dir="${repo_dir//.git}"
 cd "${repo_dir}"
 git fetch || true
 git checkout "${cloud_hypervisor_version}"
-"${script_dir}/docker-build/build.sh"
+./scripts/dev_cli.sh build --release --libc musl
 rm -f cloud-hypervisor
-cp ./target/release/cloud-hypervisor .
+cp build/cargo_target/x86_64-unknown-linux-musl/release/cloud-hypervisor .
```

Comment:
I see that the docker file is removed below. Are the builds no longer performed inside a container?

Reply:
Yes, the build is inside a container, but with the `cloud-hypervisor` source folder mounted. The final build artifact (static binary) will be in that location (`build/cargo_target/x86_64-unknown-linux-musl/release/cloud-hypervisor` relative to the clh src root). I chose to keep the `cp` after the build assuming this will make this change transparent to the rest of the CI/building flow.

Reply:
Can you replace that hard-coded architecture with `uname -m`?

Reply:
@jodh-intel Good point. Updated the PR.